packages/account/src/BiconomySmartAccountV2.ts

```diff
@@ -84,6 +84,12 @@ export class BiconomySmartAccountV2 extends BaseSmartAccount {
     const instance = new BiconomySmartAccountV2(biconomySmartAccountConfig);
     instance.factoryAddress = biconomySmartAccountConfig.factoryAddress ?? DEFAULT_BICONOMY_FACTORY_ADDRESS; // This would be fetched from V2
 
+    if (biconomySmartAccountConfig.biconomyPaymasterApiKey) {
```

Comment:
in the else part here if config.paymaster is present then assign it to instance.paymaster